qubit-sdk/src/main/java/com/qubit/android/sdk/internal/eventtracker/EventTrackerImpl.java

```diff
@@ -331,6 +339,28 @@ public void run() {
       }
       scheduleNextSendEventsTask();
     }
+
+    private void storeEventsToSend(List<EventModel> eventsToSend) {
```

Comment:
Name of method is little bit misleading.

It doesn't store eventsToSent. It stores placement attributes.

Reply:
Renamed to `placementAttributesInteractor.storeEventAttribute(eventModel)` (and moved to `StoreEventTask`)